### ISSUE TITLE
How to export pytorch crnn model to onnx?

### ISSUE THREAD

**Issue by nissansz** [sartaj0/TextRecognition-Pytorch#1]
How to export pytorch crnn model to onnx?

**Comment by sartaj0** [comment]
Hello @nissansz 
Once the training is complete pipeline will export best model to onnx format

Also you can you use the below code 
`dummy_input = torch.randn(1, imgChannel, img_height, img_width)`
`torch.onnx.export(crnn, dummy_input, "filename.onnx"), verbose=True)`

**Comment by nissansz** [comment]
Thank you. I referred to paddle2onnx to add a softmax layer, now it works.

By the way, do you know reasons for below?

genereate 28 images
separately with my own code:  6  seconds. With modified code from other github module: 2 seconds
But when it is put into RecDataSet, my own code is several times faster than  modified external module.
How to improve time other put it in RecDataSet?
Need to put modified code in RecDataSet? Or create a seperate class?
